lib/core/src/Cardano/Pool/Metadata.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+
+module Cardano.Pool.Metadata
```

Comment:
A comment explaining some basics would be nice. Might be useful to answer *any* or multiple of:
- What is a Metadata Aggregation Server?
- Any urls where I can read more about them? (ok, I see https://jira.iohk.io/browse/CAD-736, but not public)
- Any haddock items I should check out in particular? (`StakePoolOffChainMetadata`? `Client`?)

Also, just curious: is it known whether the Voltaire-testnet will use these servers, or a repo like ITN?

Reply:
Maybe https://jira.iohk.io/browse/CAD-736 could be linked regardless with a TODO to replace with a public link if/once it exists?

Reply:
> A comment explaining some basics would be nice

Indeed! Will add.

> Any urls where I can read more about them?

Actually, we could link to: https://github.com/input-output-hk/smash/

> Is it known whether the Voltaire-testnet will use these servers, or a repo like ITN?

That is a good question. I am still unsure about the scope of the Voltaire testnet but the direction it seems to take is that it'll be an extension of the ITN and therefore, the old metadata registry on the CF repository still applies.

Reply:
8b10ed9c6de3b7f828716a9e453a0f4129a281b0